src/components/navigation/QuizConceptionView.js

```diff
@@ -0,0 +1,38 @@
+import Box from "@mui/material/Box";
+import {Tab, Tabs} from "@mui/material";
+import {useState} from "react";
+import CreateView from "../create/CreateView";
+import ResultTables from "../results/ResultTables";
+
+const QuizConceptionView = () => {
+
+  const [tab, setTab] = useState(0)
+
+  const TabPanel = ({children, tab, index}) => {
+    return (
+        tab === index && (
+            <Box sx={{pt: 4}}>
+              {children}
+            </Box>)
+    )
+  }
```

Comment:
The indents is not very consistent, I would suggest you to run `yarn prettier:write` and/or have the prettier plugin for your editor to run automatically on save!

Reply:
Ok thanks, I didn't know about `yarn prettier:write`. I ran the code formatter of Intellij directly, but it seems like it is not configured the same as `prettier`.

I will use that from now on.